CMakeExt/CompilerFlags.cmake

```diff
@@ -170,9 +170,9 @@ set(CMAKE_CXX_FLAGS_RELEASE
     "${CMAKE_CXX_FLAGS_RELEASE} ${CXX_ENV_SETUP_FLAGS}")
 
 set(CMAKE_C_FLAGS_DEBUG
-    "${CMAKE_C_FLAGS_DEBUG} ${CC_STD_FLAG} ${CXX_OMP_FLAG} ${CC_REPORT_FLAG} ${CC_WARN_FLAG} -O0 -DDASH_DEBUG ${CC_GDB_FLAG}")
+    "${CMAKE_C_FLAGS_DEBUG} ${CC_STD_FLAG} ${CXX_OMP_FLAG} ${CC_REPORT_FLAG} ${CC_WARN_FLAG} -Ofast -DDASH_DEBUG ${CC_GDB_FLAG}")
```

Comment:
It's on purpose. I had no trouble with -Ofast in gdb 7.x for icc 17 and gcc 5-6 but failed to reproduce some aliasing-related bugs without it. I did not observe any advantage with -Og. gdb even kindly annotates variables with <optimized out> for -Ofast, extremely useful!